### PR TITLE
Bug 1207471 - reorganized shims in alphabet order, r=yzen

### PR DESCRIPTION
**Pull request by gasolin**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ae429f80c6d0a546fd7fa316c6514356301da679

**Comment by gasolin** [comment]
`simcard_lock` is removed so there are extra 3 lines deleted

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=370184e2d4627290407f8855e2b7373c7b38227b

File: apps/settings/build/settings.build.jslike [hunk 1]
```diff
@@ -9,40 +9,41 @@
   // which makes the objects inaccessible by reference.
   // If the inquiries to the object are all performed by requirejs, we can
   // remove the path of the object from the following list.
+  // organize in alphabet order
```
Comment (yzen) [inline apps/settings/build/settings.build.jslike]: nit: here and the other file
organize -> organized

**Comment by yzen** [comment]
I suspect it's unrelated but while stepping into panels when testing i get the following:
TypeError: iccCard is null carrier.js:174:9 - stepping into Cellular & Data with no SIM

File: apps/settings/build/settings.build.jslike [hunk 2]
```diff
@@ -9,40 +9,41 @@
...
   paths: {
     'vendor/jszip': 'empty:',
```
Comment (yzen) [inline apps/settings/build/settings.build.jslike]: A suggestion here, lets also alphabetize here based on the first character of the parent folder (not just file name). So for example modules/... will go before venoder/.... In fact vendor should go closer to the end.

Comment (gasolin) [inline apps/settings/build/settings.build.jslike]: It definitely a bug, thanks for catching it!

**Comment by gasolin** [comment]
`iccCard is null` is unrelated bug its a bug
https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/js/carrier.js#L174

filed in bug 1216847

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=40c6f86a5829f0893c90f6e3403ed81c244a6f4f

**Pull request merged**